people-and-planet-ai/image-classification/README.ipynb

```diff
@@ -0,0 +1,808 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Monitor wildlife health via image classification",
+      "private_outputs": true,
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3",
+      "language": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "code",
```

Comment:
This cell runs, but it's just the ASF licensing.

Reply:
Yeah, the license is required by the linters for all files.